audplot/core/api.py

```diff
@@ -8,6 +9,73 @@
 import audmetric
 
 
+def cepstrum(
+        cc_matrix: np.ndarray,
+        sampling_rate: float,
+        *,
+        channel: int = 0,
+        ax: plt.Axes = None,
+        cmap: str = 'magma',
+) -> matplotlib.image.AxesImage:
+    r"""Cepstrum.
+
+    Args:
+        cc_matrix: cepstral coefficients matrix with magnitude values
+        sampling_rate: sampling rate in Hz
+        channel: channel index
+        ax: axes in which to draw the plot
+        cmap: color map
+
+    Returns:
+        Image object
+
+    Example:
+        .. plot::
+            :context: reset
+            :include-source: false
+
+            from audplot import cepstrum
+
+        .. plot::
+            :context: close-figs
+
+            >>> import librosa
+            >>> import matplotlib.pyplot as plt
+            >>> x, sr = librosa.load(librosa.ex('trumpet'))
+            >>> y = librosa.feature.mfcc(x, sr)
+            >>> y_sr = sr / 512  # default hop length is 512
```

Comment:
OK, now I see what you mean. I don't think a user will ever realize that she has to do this.
So the advantage of giving duration was that you could calculate this without knowing the hop duration.

Maybe it will be more obvious if we add `hop_duration` as another argument?

Or at least we have to update the description of sampling rate, stating that it is not the sampling rate of the original signal.

Reply:
Yes, and the problem with `duration` is that then it's not obvious how to rename the ticks (since we don't have the `sampling_rate`) and that was the reason to not use the default ticks but customize via `num_ticks`.

But maybe `hop_duration` is indeed a good compromise.

Reply:
Switched to `hop_duration` now, hopefully the last time I had to update the plots in the description :)

Reply:
Cool, looks good now.